addons/stock/models/stock_picking.py

```diff
@@ -515,7 +515,7 @@ def _compute_show_check_availability(self):
     @api.depends('state', 'move_lines')
     def _compute_show_mark_as_todo(self):
         for picking in self:
-            if not picking.move_lines and not picking.package_level_ids:
+            if not picking.package_level_ids and picking.immediate_transfer:
```

Comment:
Why do you need this change?

Reply:
yes, its working without this change. Removing it.